### PR TITLE
docs: provide example for enabling mesh on a per namespace basis

### PR DESCRIPTION
**Pull request by david-yu**
Provide a more explicit example for enabling mesh by using a label.

### CODE REVIEW

File: website/content/docs/k8s/installation/install.mdx [hunk 1]
```diff
@@ -175,8 +175,9 @@ create a `config.yaml` file to override the default settings.
 You can learn what settings are available by running `helm inspect values hashicorp/consul`
 or by reading the [Helm Chart Reference](/docs/k8s/helm).
 
-For example, if you want to enable the [Consul Connect](/docs/k8s/connect) feature,
-use the following config file:
+#### Minimal `values.yaml` for Consul Service Mesh
+
+The minimal settings to enable [Consul Service Mesh]((/docs/k8s/connect)) would be captured in the following `vaules.yaml` config file:
```
Comment (blake) [inline website/content/docs/k8s/installation/install.mdx:180]:
```suggestion
#### Minimal `config.yaml` for Consul Service Mesh

The minimal settings to enable [Consul Service Mesh]((/docs/k8s/connect)) would be captured in the following `config.yaml` config file:
```
The code blocks below use `config.yaml` as the filename.

File: website/content/docs/k8s/installation/install.mdx [hunk 2]
```diff
@@ -199,6 +200,44 @@ NAME: consul
 ...
 ```
 
+#### Enable Consul Service Mesh on select namespaces
+
+By default, Consul Service Mesh is enabled on all namespaces within Kubernetes. However if you'd like to control which namespaces allow connect-inject to occur, you can alternatively use a label on a namespace to denote where to enable Consul Service Mesh. 
+
```
Comment (blake) [inline website/content/docs/k8s/installation/install.mdx:206]:
```suggestion
By default, Consul Service Mesh is enabled on all namespaces within a Kubernetes cluster. You can restrict this to a subset of namespaces by specifying a `namespaceSelector` that matches a label attached to each namespace denoting whether to enable Consul service mesh.
```

**Review by t-eckert — APPROVED** [review]

File: website/content/docs/k8s/installation/install.mdx [hunk 3]
```diff
@@ -199,6 +200,43 @@ NAME: consul
 ...
 ```
 
+#### Enable Consul Service Mesh on select namespaces
+
+By default, Consul Service Mesh is enabled on all namespaces within a Kubernetes cluster. You can restrict this to a subset of namespaces by specifying a `namespaceSelector` that matches a label attached to each namespace denoting whether to enable Consul service mesh.
```
Comment (ishustava) [inline website/content/docs/k8s/installation/install.mdx:205]: We exclude a couple by default: https://github.com/hashicorp/consul-k8s/blob/5027066696e2a95fea6342da96081c1581574149/charts/consul/values.yaml#L1797
```suggestion
By default, Consul Service Mesh is enabled on almost all namespaces within a Kubernetes cluster. You can restrict this to a subset of namespaces by specifying a `namespaceSelector` that matches a label attached to each namespace denoting whether to enable Consul service mesh.
```

File: website/content/docs/k8s/installation/install.mdx [hunk 4]
```diff
@@ -199,6 +200,43 @@ NAME: consul
...
+<CodeBlockConfig filename="config.yaml">
+
+```yaml
+global:
+  name: consul
+connectInject:
+  enabled: true
+  default: true
```
Comment (ishustava) [inline website/content/docs/k8s/installation/install.mdx:213]: Should we call this out a bit more that setting `default` to `true` is required for this to work?

**Review by ishustava — APPROVED** [review]
Looks great! I left a couple suggestions

File: website/content/docs/k8s/installation/install.mdx (hunk 4 above)
Comment (david-yu) [reply website/content/docs/k8s/installation/install.mdx:214]: Thanks added!

**Comment by hc-github-team-consul-core** [comment]
:cherries: If backport labels were added before merging, cherry-picking will start automatically.

To retroactively trigger a backport after merging, add backport labels and re-run https://circleci.com/gh/hashicorp/consul/574933.

**Comment by hc-github-team-consul-core** [comment]
:cherries::white_check_mark: Cherry pick of commit 81461565d7565a3e6b5a1ab1c1e7d04d5b3d2a19 onto `stable-website` succeeded!

**Comment by hc-github-team-consul-core** [comment]
:cherries::white_check_mark: Cherry pick of commit 81461565d7565a3e6b5a1ab1c1e7d04d5b3d2a19 onto `release/1.11.x` succeeded!

**Pull request merged**